account_prefs_update.php

```diff
@@ -139,10 +139,5 @@
 
 form_security_purge( 'account_prefs_update' );
 
-layout_page_header( null, $f_redirect_url );
+print_header_redirect( $f_redirect_url, /* force_show */ true );
```

Comment:
I think this is wrong - print_header_redirect()'s second parameter is not *force_show*... if true, it stops the script after redirecting.

The 2nd parameter should be dropped.